test/index.html

```diff
@@ -35,7 +35,7 @@
 			amd = QUnit.config.amd || QUnit.urlParams.amd;
 
 		// Workaround: Remove call to `window.__karma__.loaded()`
-		// in favour of calling `window.__karma__.start()` from `loadTests()`
```

Comment:
It's not incorrect, this is just British spelling. I think in principle we follow American English, though, so it looks like a good change.